uwp/xbox-apps/devkit-activation.md

```diff
@@ -1,73 +1,73 @@
 ---
-title: Xbox One Developer Mode activation
+title: Xbox Developer Mode activation
 description: How to activate Developer Mode so you can switch between Retail Mode and Developer Mode.
-ms.date: 02/08/2017
+ms.date: 09/23/2023
 ms.topic: article
 keywords: windows 10, uwp
 ms.assetid: ade80769-17ae-46e9-9c2f-bf08ae5a51ee
 ms.localizationpriority: medium
 ---
-# Xbox One Developer Mode activation
+# Xbox Developer Mode activation
 
 ## How Developer Mode works
-This article only applies to Xbox One and Xbox Series X | S consoles acquired through retail channels. For development kit HW acquired via a managed development program, see the note at the end of the article.
+This article only applies to Xbox One and Xbox Series X|S consoles acquired through retail channels. For development kit HW acquired via a managed development program, see the note at the end of the article.
 
-Xbox retail consoles can have two modes, Retail Mode (1) and Developer Mode (2). In Retail Mode, the console is in its normal state: you can play games and run apps acquired through the Xbox store. In Developer Mode, you can develop and test software for the console, but you cannot play retail games or run retail apps.
+Xbox retail consoles can have two modes, Retail Mode (1) and Developer Mode (2). In Retail Mode, the console is in its normal state: you can play games and run apps acquired through the Microsoft Store. In Developer Mode, you can develop and test software for the console, but you cannot play retail games or run retail apps.
 
-Developer Mode can be enabled on any retail Xbox console, via the “Retail to Dev Kit conversion” app found in the Xbox store. After Developer Mode is enabled on your retail console, you can switch back and forth between Retail (2a) and Developer Modes (2b).
+Developer Mode can be enabled on any retail Xbox console, via the [Xbox Dev Mode](https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) app found in the Microsoft Store. After Developer Mode is enabled on your retail console, you can switch back and forth between Retail (2a) and Developer Modes (2b).
```

Comment:
I'd suggest using this link instead (remove the en-us from the path) - https://www.microsoft.com/p/xbox-dev-mode/9nljhzjrn0f4

Reply:
For sure. Done.